pkg/capabilities/capabilities.go

```diff
@@ -367,3 +367,35 @@ outerLoop:
 
 	return &values.List{Underlying: vs}, nil
 }
+
+const (
+	DefaultRegistrationRefreshMs = 30_000
+	DefaultRegistrationExpiryMs  = 120_000
+	DefaultMessageExpiryMs       = 120_000
```

Comment:
Would `time.Duration`s be easier to work with?
```suggestion
	DefaultRegistrationRefresh = 30 * time.Second
	DefaultRegistrationExpiry  = 2 * time.Minute
	DefaultMessageExpiry       = 2 * time.Minute
```